src/platform/EFR32/CHIPDevicePlatformConfig.h

```diff
@@ -48,17 +48,15 @@
 
 #define CHIP_DEVICE_CONFIG_ENABLE_CHIP_TIME_SERVICE_TIME_SYNC 0
 
-#if defined(RS911X_WIFI)
+#if defined(SL_WIFI)
 
 #if defined(WIFI_IPV4_DISABLED)
 #define CHIP_DEVICE_CONFIG_ENABLE_IPV4 0
-#define CHIP_DEVICE_CONFIG_ENABLE_IPV6 1
-#else
+#else /* !WIFI_IPV4_DISABLED */
 #define CHIP_DEVICE_CONFIG_ENABLE_IPV4 1
-#define CHIP_DEVICE_CONFIG_ENABLE_IPV6 1
-#endif
+#endif /* WIFI_IPV4_DISABLED */
 
-#endif
+#endif /* SL_WIFI */
 
```

Comment:
cool but where SL_WIFI is defined now ? directly into your SDK itself ? (third_party/efr32_sdk ?)

Reply:
Yeah its set in the GNI files, `SL_WIFI=1` for both WiFi modules